src/components/Inputs/CustomNumberField.js

```diff
@@ -84,11 +87,15 @@ const CustomNumberField = ({
       onInput={handleInput}
       InputProps={{
         inputProps: {
-          tabIndex: readOnly ? -1 : 0 // Prevent focus on the input field
+          min: min,
+          max: max,
+          step: step,
```

Comment:
what does step stand for here?

Reply:
when  you decrease the value it decrease by step = 1